### PR TITLE
docs: Add CONTRIBUTING.md file to the Kedro-Plugins repository

### PR DESCRIPTION
**Pull request by lrcouto**
## Description

Part of the solution for issue [#2667](https://github.com/kedro-org/kedro/issues/2667). Adds a `CONTRIBUTING.md` file to the Kedro-Plugins repository, linking to the contributing guide on the wiki.

## Checklist

- [x] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [x] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the relevant `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

**Review by noklam — COMMENTED** [review]
Is this identical to the kedro contributor.md now?


https://github.com/kedro-org/kedro/issues/2667#issuecomment-1587351366

Is this comment addressed?

**Comment by lrcouto** [comment]
At this moment, yes, it's identical to the file on the Kedro repo.

I've added a "Which contributions are likely to be accepted?" section to the wiki page to address Merel's comment.

**Review by stichbury — CHANGES_REQUESTED** [review]
I suggest for now that you take the version you ultimately merge from https://github.com/kedro-org/kedro/pull/2894 and use this, but also take this on board from @merelcht and add a section specific to the `kedro-plugins` repo.

**Comment by merelcht** [comment]
I think this looks good, but as @stichbury said above it would be good to add a section about `kedro-plugins` contributions specifically similar to https://github.com/kedro-org/kedro/wiki/Contribute-to-Kedro#which-contributions-are-likely-to-be-accepted

**Review by merelcht — APPROVED** [review]
LGTM! 👍

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Introduction
+
+We welcome any and all contributions to Kedro, at whatever level you can manage. Here are a few suggestions, but you are welcome to suggest anything else that you think improves the community for us all!
```
Comment (astrojuanlu) [inline CONTRIBUTING.md:3]:
```suggestion
We welcome any and all contributions to Kedro, at whatever level you can manage. Here are a few suggestions, and you are welcome to suggest anything else that you think improves the community for us all!
```

small change, but no need for an adversative word there? 😊

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## Join the community
+
+You can find the Kedro community on our [Slack organisation](https://slack.kedro.org/), which is where we share news and announcements, and answer technical questions. You're welcome to post links to any articles or videos about Kedro that you create or find, such as how-tos, showcases, demos, blog posts or tutorials.
+
+We also curate a [GitHub repo that lists content created by the Kedro community](https://github.com/kedro-org/awesome-kedro). If you've made something with Kedro, simply add it to the list with a PR!
+
+## Contribute to the project
+
+There are quite a few ways to contribute to Kedro, sich as answering questions about Kedro to help others, fixing a typo on the documentation, reporting a bug, reviewing pull requests or adding a feature.
```
Comment (astrojuanlu) [inline CONTRIBUTING.md:13]:
```suggestion
There are quite a few ways to contribute to Kedro, such as answering questions about Kedro to help others, fixing a typo on the documentation, reporting a bug, reviewing pull requests or adding a feature.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+Take a look at some of our [contribution suggestions on the Kedro GitHub Wiki](https://github.com/kedro-org/kedro/wiki/Contribute-to-Kedro)!
+
+## Which plugin contributions are likely to be accepted?
+
+Dataset contributions to the [Kedro-Datasets](https://github.com/kedro-org/kedro-plugins/tree/main/kedro-datasets) plugin are the most frequently accepted, since they do not require any changes to the framework itself. 
+
+However, we do not discourage contributions to any of the other [Kedro-Plugins](https://github.com/kedro-org/kedro-plugins) or the framework or `Kedro-Viz`. As a guide, contributions based on existing issues from the Kedro team, or issues that the team has deemed useful, are most likely to be accepted. Any contributions that affect fundamental changes to the Kedro Framework would require discussion first. In this case, we recommend opening an issue instead of a pull request.
```
Comment (astrojuanlu) [inline CONTRIBUTING.md:21]: Maybe instead of "we do not discourage", just "we accept"?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+Make sure to check out the contributing guides for [Kedro-Docker](https://github.com/kedro-org/kedro-plugins/blob/main/kedro-docker/CONTRIBUTING.md) and [Kedro-Airflow](https://github.com/kedro-org/kedro-plugins/blob/main/kedro-airflow/CONTRIBUTING.md) if you intend to contribute to those specific plugins.
+
+## Join the Technical Steering Committee
+Kedro is an incubating project in [LF AI & Data](https://lfaidata.foundation/), a sub-organisation within the Linux Foundation that focuses on open innovation within the data and AI space.
+
+The project is governed by a group of maintainers, known as the Technical Steering Committee (TSC); read more about the structure of our TSC in our [Technical Charter](./kedro_technical_charter.pdf).
+
+We regularly invite community members to join the TSC and help define the future of the Kedro project. Read the [guidance on becoming a Kedro maintainer](https://docs.kedro.org/en/stable/contribution/technical_steering_committee.html) to understand the process of joining the TSC.
+
+## Code of conduct
```
Comment (astrojuanlu) [inline CONTRIBUTING.md:32]: Can we maybe move this to the top? Right after the intro

**Review by astrojuanlu — APPROVED** [review]
Looks wonderful! Left some nitpicks and comments

File: CONTRIBUTING.md (hunk 3 above)
Comment (lrcouto) [reply CONTRIBUTING.md]: Done!

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -15,12 +15,12 @@ We also curate a [GitHub repo that lists content created by the Kedro community]
 ## Contribute to the project
 
 There are quite a few ways to contribute to Kedro, such as answering questions about Kedro to help others, fixing a typo on the documentation, reporting a bug, reviewing pull requests or adding a feature.
-
+ls
```
Comment (astrojuanlu) [inline CONTRIBUTING.md:18]: Typo!

**Review by stichbury — APPROVED** [review]
LGTM 🌟

**Pull request merged**